packages/normalizr/src/denormalize.ts

```diff
@@ -21,72 +32,131 @@ const unvisitEntity = (
     return [entity, false, false];
   }
 
-  if (!cache[schema.key]) {
-    cache[schema.key] = {};
+  if (!localCache[schema.key]) {
+    localCache[schema.key] = {};
   }
```

Comment:
if `schema.key` could be any string (or Symbol) value, then you may want to use another approach so a key of, say, `toString` doesn't return the wrong value.

Reply:
would `!== undefined` be just as good vs the prototype.call? Just to save some bytes

Reply:
only as long as nobody's done `Object.prototype.something = undefined`, but i suppose in that case it would still work fine.